sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/HistogramAccumulation.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.aggregator;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.api.common.Labels;
+import io.opentelemetry.sdk.metrics.common.ImmutableDoubleArray;
+import io.opentelemetry.sdk.metrics.common.ImmutableLongArray;
+import io.opentelemetry.sdk.metrics.data.DoubleHistogramPointData;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+@AutoValue
+public abstract class HistogramAccumulation {
```

Comment:
A bit of explanation about `Accumulation` that may help to understand why boundaries don't need to be here.

The framework that we build and uses the Aggregator guarantees that an instance of the `Accumulation` that is created by a specific instance of an Aggregator will be `merged` and converted to metrics by the same specific Aggregator instance.

So as long as one Aggregator instance produces all Accumulations with constant boundaries you don't need to worry about merging accumulations with different boundaries.